packages/camera-web/src/Camera/hooks/useUserMedia.ts

```diff
@@ -36,6 +36,15 @@ class InvalidStreamError extends Error {
  * @see useUserMedia
  */
 export enum UserMediaErrorType {
+  /**
+   * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
+   * If both webpage and browser permissions are denied, this error will be returned.
```

Comment:
Change description to :

```
/**
 * The camera stream couldn't be fetched because the web page does not have the permissions to access the camera.
 */
```